src/compiler/diagnosticMessages.json

```diff
@@ -683,6 +683,14 @@
         "category": "Error",
         "code": 1218
     },
+    "Type declaration expected.": {
+        "category": "Error",
+        "code": 1219
+    },
+    "Can't define a type-guard type inside a type-guard type.": {

```

Comment:
Will we eventually only allow a type guard type (type predicate) in return position? If so, I think this restriction is just a special case of that, no?

I would use "cannot" instead of "can't". Just for style.